### PR TITLE
DATAUP-678: minor job manager edits

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Extremely minor PR, reordering methods in the py job manager and changing the names of some methods to be `get(_all)*_job` instead of `lookup(_all)*_job` for internal consistency in the module.

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/DATAUP-678
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (Python) I have run Black and Flake8 on changed Python code manually or with a git precommit hook

### CODE REVIEW

File: src/biokbase/narrative/jobs/jobcomm.py [hunk 1]
```diff
@@ -171,13 +171,13 @@ def __init__(self):
         if self._msg_map is None:
             self._msg_map = {
                 MESSAGE_TYPE["CANCEL"]: self._cancel_jobs,
-                MESSAGE_TYPE["CELL_JOB_STATUS"]: self._lookup_job_states_by_cell_id,
+                MESSAGE_TYPE["CELL_JOB_STATUS"]: self._get_job_states_by_cell_id,
```
Comment (ialarmedalien) [inline src/biokbase/narrative/jobs/jobcomm.py:174]: lookup => get  -- to make all method names consistent

File: src/biokbase/narrative/jobs/jobmanager.py [hunk 2]
```diff
@@ -264,31 +264,30 @@ def _construct_job_output_state_set(
 
         return output_states
 
-    def get_job_info(self, job_ids: List[str]) -> dict:
+    def get_job_states(self, job_ids: List[str]) -> dict:
```
Comment (ialarmedalien) [inline src/biokbase/narrative/jobs/jobmanager.py:267]: moved

File: src/biokbase/narrative/jobs/jobmanager.py [hunk 3]
```diff
@@ -264,31 +264,30 @@ def _construct_job_output_state_set(
...
+        job_ids, error_ids = self._check_job_list(job_ids)
+        output_states = self._construct_job_output_state_set(job_ids)
+        return self.add_errors_to_results(output_states, error_ids)
+
+    def get_all_job_states(self, ignore_refresh_flag=False) -> dict:
```
Comment (ialarmedalien) [inline src/biokbase/narrative/jobs/jobmanager.py:272]: moved and renamed

File: src/biokbase/narrative/jobs/jobmanager.py [hunk 4]
```diff
@@ -308,7 +307,7 @@ def _get_job_ids_by_cell_id(self, cell_id_list: List[str] = None) -> tuple:
         job_id_list = set().union(*cell_to_job_mapping.values())
         return (job_id_list, cell_to_job_mapping)
 
-    def lookup_job_states_by_cell_id(self, cell_id_list: List[str] = None) -> dict:
+    def get_job_states_by_cell_id(self, cell_id_list: List[str] = None) -> dict:
```
Comment (ialarmedalien) [inline src/biokbase/narrative/jobs/jobmanager.py:310]: renamed

File: src/biokbase/narrative/jobs/jobmanager.py [hunk 5]
```diff
@@ -323,25 +322,31 @@ def lookup_job_states_by_cell_id(self, cell_id_list: List[str] = None) -> dict:
 
         return {"jobs": job_states, "mapping": cell_to_job_mapping}
 
-    def lookup_all_job_states(self, ignore_refresh_flag=False) -> dict:
+    def get_job_info(self, job_ids: List[str]) -> dict:
```
Comment (ialarmedalien) [inline src/biokbase/narrative/jobs/jobmanager.py:325]: moved

**Review by briehl — APPROVED** [review]
Yup, looks pretty minor to me!

**Pull request merged**